src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -41,7 +41,6 @@ public void Dispose()
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "https://github.com/dotnet/corefx/issues/19379")]
```

Comment:
Should this PR be "linked" to #19379 so that whoever looks at that issue can see that some progress is being made?